src/main/java/org/reactome/addlinks/EnsemblBioMartUtil.java

```diff
@@ -0,0 +1,160 @@
+        try(FileInputStream fis = new FileInputStream(propertiesLocation))
+        {
+            applicationProps.load(fis);
+        }
+        return applicationProps;
+    }
+
+    /**
+     * Returns data corresponding to 'species_uniprotToProteins' in mappings.
+     * @param speciesBiomartName - String, BioMart-formatted species name
+     * @param mappings- Map<String, Map<String, List<String>>>, Mapping generated from EnsemblBioMartFileProcessor.
+     * @return Map<String, List<String>>, corresponding mappings for species.
+     */
+    public static Map<String, List<String>> getUniprotToProteinsMappings(String speciesBiomartName, Map<String, Map<String, List<String>>> mappings) {
+        String uniprotToProteinsKey = speciesBiomartName + UNIPROT_TO_PROTEINS_SUFFIX;
+        return mappings.computeIfAbsent(uniprotToProteinsKey, k -> new HashMap<>());
+    }
+
+    /**
+     * Returns data corresponding to 'species_transcriptToMicroarrays' in mappings.
+     * @param speciesBiomartName - String, BioMart-formatted species name
+     * @param mappings- Map<String, Map<String, List<String>>>, Mapping generated from EnsemblBioMartFileProcessor.
+     * @return Map<String, List<String>>, corresponding mappings for species.
+     */
+    public static Map<String, List<String>> getTranscriptToMicroarraysMappings(String speciesBiomartName, Map<String, Map<String, List<String>>> mappings) {
+        String transcriptToMicroarraysKey = speciesBiomartName + TRANSCRIPT_TO_MICROARRAYS_SUFFIX;
+        return mappings.computeIfAbsent(transcriptToMicroarraysKey, k -> new HashMap<>());
+    }
+
+    /**
+     * Returns data corresponding to 'species__proteinToTranscripts' in mappings.
+     * @param speciesBiomartName - String, BioMart-formatted species name
+     * @param mappings- Map<String, Map<String, List<String>>>, Mapping generated from EnsemblBioMartFileProcessor.
+     * @return Map<String, List<String>>, corresponding mappings for species.
+     */
+    public static Map<String, List<String>> getProteinToTranscriptsMappings(String speciesBiomartName, Map<String, Map<String, List<String>>> mappings) {
+        String proteinToTranscriptsKey = speciesBiomartName + PROTEIN_TO_TRANSCRIPTS_SUFFIX;
+        return mappings.computeIfAbsent(proteinToTranscriptsKey, k -> new HashMap<>());
+    }
+
+    /**
+     * Returns all lines in the file. If the file contains a single-line header, it is filtered out.
+     * @param inputFilePath - Path, location of file to be read.
+     * @param skipHeader - boolean, determines if first line of file should be filtered out.
+     * @return List<String>, contents of file in a List.
+     * @throws IOException - Thrown if file does not exist.
+     */
+    public static List<String>  getLinesFromFile(Path inputFilePath, boolean skipHeader) {
+        try {
+            List<String> fileLines = Files.readAllLines(inputFilePath, StandardCharsets.ISO_8859_1);
```

Comment:
Should the Charset be UTF_8 instead of ISO_8859_1?

Reply:
Generally I would agree, but maybe with some files it needs to be ISO_8859_1. I'd wait to hear from @cookersjs.

Reply:
Yea it turns out that the RGD file downloaded needs to be read in ISO_8859_1. Both default and UTF-8 do not work when read in using `readAllLines` for it

Reply:
Cool. Maybe just put a comment about it above that line. Many people expect UTF-* by default, and seeing something else can raise a red flag.